app/src/main/java/org/thoughtcrime/securesms/database/MmsDatabase.kt

```diff
@@ -306,48 +262,30 @@ class MmsDatabase(context: Context, databaseHelper: SQLCipherOpenHelper) : Messa
         }
     }
 
-    fun markAsPendingInsecureSmsFallback(messageId: Long) {
-        val threadId = getThreadIdForMessage(messageId)
+    private fun markAs(
```

Comment:
new courtesy method for deduplication and usage in future PR for adding syncing state.